From 620aafe7f0e9966121db387da459d12834850a24 Mon Sep 17 00:00:00 2001
From: Yuwei Chen <yuwei.chen@intel.com>
Date: Fri, 24 Jul 2020 10:21:00 +0800
Subject: [PATCH] Test case for Library - CpuSleep.asm file

Description: Change the READONLY to READWRITE in the MdePkg/Library/BaseCpuLib/Arm/CpuSleep.asm.

Repo: edk2
Signed-off-by: Yuwei Chen <yuwei.chen@intel.com>
---
 MdePkg/Library/BaseCpuLib/Arm/CpuSleep.asm | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/MdePkg/Library/BaseCpuLib/Arm/CpuSleep.asm b/MdePkg/Library/BaseCpuLib/Arm/CpuSleep.asm
index ae32705a69..de238d8f9e 100644
--- a/MdePkg/Library/BaseCpuLib/Arm/CpuSleep.asm
+++ b/MdePkg/Library/BaseCpuLib/Arm/CpuSleep.asm
@@ -15,7 +15,7 @@
 ;------------------------------------------------------------------------------
 
   EXPORT CpuSleep
-  AREA cpu_sleep, CODE, READONLY
+  AREA cpu_sleep, CODE, READWRITE
 
 ;/**
 ;  Places the CPU in a sleep state until an interrupt is received.
-- 
2.27.0.windows.1

